.github/workflows/ci.yml

```diff
@@ -684,7 +684,7 @@ jobs:
           - { sample: scala-protobuf-view-store, test: true }
 
           - { sample: scala-protobuf-eventsourced-customer-registry, test: true }
-          - { sample: scala-protobuf-eventsourced-customer-registry-subscriber, test: true }
+          - { sample: scala-protobuf-eventsourced-customer-registry-subscriber, pre_cmd: 'cd scala-protobuf-eventsourced-customer-registry; sbt -Dkalix-sdk.version=${SDK_VERSION} docker:publishLocal; cd -', test: true }
```

Comment:
This was not working for maven and will probably not work here either.

I still need to find a workaround for it.

Reply:
Oh, and know I see that I missed a `../`